.github/changelog.yml

```diff
@@ -0,0 +1,3 @@
+filename: CHANGELOG.md
+include:
+  - /^bin\/|^config\/|^scripts\/|^src\/|^package.json/
```

Comment:
Should these be updated to match this repo structure?

Reply:
yes

Reply:
I'm deleting this for now because we have multiple change logs